Desktop/mainwindow.cpp

```diff
@@ -168,7 +168,8 @@ MainWindow::MainWindow(QApplication * application) : QObject(application), _appl
 	qmlRegisterType<JASPDoubleValidator>						("JASP",		1, 0, "JASPDoubleValidator"				);
 	qmlRegisterType<ResultsJsInterface>							("JASP",		1, 0, "ResultsJsInterface"				);
 
-	qmlRegisterUncreatableType<PlotEditor::AxisModel>			("JASP.PlotEditor",		1, 0, "AxisModel",		"Can't make it");
+	qmlRegisterUncreatableType<PlotEditor::AxisModel>			("JASP.PlotEditorAxis",		1, 0, "AxisModel",			"Can't make it");
```

Comment:
Isn't it more logical to put both of these QML items inside the same group called "JASP.PlotEditor" ?

That way you can just do `import JASP.PlotEditor 1.0` and then use `AxisModel` and `PlotEditorModel` directly.
The way you've done it now is rather confusing.

Reply:
that somehow didn't work before, but I'll try again :)

Reply:
and now it does work ...